src/entry-point-client/App.tsx

```diff
@@ -5,6 +5,7 @@ import { Route, Switch } from "react-router-dom";
 import AuthWall from "../auth/AuthWall";
 import { LocaleDataProvider } from "../locale-data-context";
 import { FacilityContext } from "../page-multi-facility/FacilityContext";
+import { ScenarioProvider } from "../scenario-context";
```

Comment:
Now that we'll be able to toggle between scenarios, we'll need to keep track of the information about which scenario we're working within more globally.  This is very similar to the work we've already done at the Facility level with `FacilityContext`, however, I've started to use the more "provider" pattern as illustrated in [LocaleDataContext](https://github.com/Recidiviz/covid19-dashboard/blob/master/src/locale-data-context/LocaleDataContext.tsx) and [EpidemicModelContext](https://github.com/Recidiviz/covid19-dashboard/blob/master/src/impact-dashboard/EpidemicModelContext.tsx).

In the near future, I'll update FacilityContext to follow this pattern as well